File: Python/Fixed_Dynamic.py
fixed_array = [0] * 5 

def display_array():
    print("Fixed Dynamic Array:", fixed_array)

def update_element(index, value):
    if 0 <= index < len(fixed_array):
        fixed_array[index] = value
    else:
        print("Index out of bounds.")

if __name__ == "__main__":
    display_array()
    update_element(2, 10)
    display_array()

File: CPP/Fixed_Dynamic.cpp
#include <iostream>
using namespace std;

int main() {
    int fixedArray[5] = {0};  // Size fixed at compile time

    cout << "Fixed Dynamic Array: ";
    for (int i = 0; i < 5; i++)
        cout << fixedArray[i] << " ";
    cout << endl;

    fixedArray[2] = 10;
    cout << "After updating index 2: ";
    for (int i = 0; i < 5; i++)
        cout << fixedArray[i] << " ";
    cout << endl;

    return 0;
}
